### ISSUE TITLE
a11y: Missing grouping of form controls. - RadioGroup

### ISSUE THREAD

**Issue by aalves08** [rancher/dashboard#13837]
**Setup**
- Rancher version:
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

**a11y issue 77**

The radio buttons are not programmatically grouped and associated with the instruction text resulting in screen reader users finding it difficult to understand the purpose of the radio buttons.

Main Content:
Login landing page
- Take me to the home page
- Take me to the area I last visited
- Take me to cluster
radio buttons 

Include the instruction "You can change where you land when you login" within `<legend>` element. Use `<fieldset>` and `<legend>` elements to group the instruction and radio buttons.

**To Reproduce**

**Result**

**Expected Result**

The screen reader should announce the instruction along with the label for the radio buttons.

**Screenshots**

**Additional context**

**Comment by izaac** [comment]
Rancher version: `v2.12-7ca04114a589de3edf62e85eded0ba89c408ec2c-head`
Rancher UI version (dashboard): `master 28636c1`

The text to speech works as expected with clear association between the option's text and the audio. 
Running the WAVE extension doesn't show error in the component, just a couple (good to have) warnings.
Checked against `v2.9.10-alpha1`

✅
